test/worldwide/region_test.rb

```diff
@@ -249,5 +249,25 @@ class RegionTest < ActiveSupport::TestCase
         assert_equal expected_country, Worldwide.region(code: region_code).associated_country.iso_code
       end
     end
+
+    test "name alternates are returned as expected" do
+      {
+        cz: ["Czechia"],
+        sz: ["Swaziland"],
+        us: [],
```

Comment:
FYI I fixed a customer-facing issue for the United States in shopify-i18n several weeks ago: https://github.com/Shopify/shopify-i18n/pull/1647

We should make sure that United States of America resolves to the US region.

Reply:
addressed